spec/cypress/support/on-rails.js

```diff
@@ -1,25 +1,30 @@
-// CypressOnRails: dont remove these command
+import { camelToSnakeCase } from '../utils'
+
 Cypress.Commands.add('appCommands', function (body) {
   cy.log('APP: ' + JSON.stringify(body))
-  return cy.request({
-    method: 'POST',
-    url: '/__cypress__/command',
-    body: JSON.stringify(body),
-    log: true,
-    failOnStatusCode: true
-  }).then((response) => {
-    return response.body
-  })
+  return cy
+    .request({
+      method: 'POST',
+      url: '/__cypress__/command',
+      body: JSON.stringify(body),
+      log: true,
+      failOnStatusCode: true
+    })
+    .then(response => {
+      return response.body
+    })
 })
 
 Cypress.Commands.add('app', function (name, command_options) {
-  return cy.appCommands({name: name, options: command_options}).then((body) => {
-    return body[0]
-  })
+  return cy
+    .appCommands({ name: camelToSnakeCase(name), options: command_options })
```

Comment:
Could you put in a comment about why this is necessary?

Reply:
Done.